.bazelrc

```diff
@@ -159,7 +159,7 @@ build:remote-msan --config=rbe-toolchain-msan
 
 # Docker sandbox
 # NOTE: Update this from https://github.com/envoyproxy/envoy-build-tools/blob/master/toolchains/rbe_toolchains_config.bzl#L7
```

Comment:
Update this description, or update the script in envoy-build-tools to also write tag in that file. (Note RBE requires sha256 as primary).

Reply:
Thanks for your comments. I will check it carefully.

Reply:
Has been updated. Thanks.